jibri/rootfs/opt/jitsi/shutdown.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/with-contenv bash
+# notify the sidecar of imminent shutdown
+[ -z "$AUTOSCALER_SIDECAR_PORT" ] && export AUTOSCALER_SIDECAR_PORT="6000"
```

Comment:
Ditto https://github.com/jitsi/docker-jitsi-meet/pull/1809/files#r1608942516 but here we call a script, do we use the env var in there?